lib/slipstream.ex

```diff
@@ -1286,10 +1285,9 @@ defmodule Slipstream do
   @spec await_reply(push_reference(), timeout()) :: reply() | {:error, :timeout}
   def await_reply(push_reference, timeout \\ @default_timeout)
 
-  def await_reply({topic, ref}, timeout) do
+  def await_reply(ref, timeout) do
     receive do
-      event(%Events.ReplyReceived{ref: ^ref, topic: ^topic} = event) ->
-        Events.ReplyReceived.to_reply(event)
```

Comment:
this and the associated change in `lib/slipstream/callback:ex:110` are a bit of a refactor: instead of rendering the reply into a common format just before passing it to the user, we do it when we initially map the `%Phoenix.Socket.Message{}` to the `%Slipstream.Events.ReplyReceived{}`

just makes it a bit more DRY and easy-to-use internally (in the library)